docs/layouts/shortcodes/generated/kubernetes_operator_config_configuration.html

```diff
@@ -236,6 +236,12 @@
             <td>Duration</td>
             <td>The timeout for the resource clean up to wait for flink to shutdown cluster.</td>
         </tr>
+        <tr>
+            <td><h5>kubernetes.operator.resource.deletion.propagation</h5></td>
+            <td style="word-wrap: break-word;">Foreground</td>
+            <td><p>Enum</p></td>
+            <td>JM/TM Resource deletion propagation.<br /><br />Possible values:<ul><li>"Orphan"</li><li>"Background"</li><li>"Foreground"</li></ul></td>
```

Comment:
nit: Technically not just the JM/TM resources, but all dependant resources including services, configmaps etc. It is still reasonably descriptive in this form. 😏